libs/ledger-live-common/src/families/polkadot/specs.ts

```diff
@@ -21,6 +21,7 @@ import {
 } from "../../families/polkadot/logic";
 import { DeviceModelId } from "@ledgerhq/devices";
 
+const maxAccounts = 8;
```

Comment:
Why 8, any reason for not using the default nb of accounts?

Reply:
As Polkadot has a 28-days unbonding period, would it make sense to increase the nb of accounts to something similar as Cosmos e.g 32 accounts? To increase chances to always have some accounts with free balance.

Reply:
yeah let's do this